
package id.co.asyst.bukopin.mobile.user.model.soap.account;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.3.4
 * 2019-10-27T21:13:21.862+07:00
 * Generated source version: 3.3.4
 */

@WebFault(name = "ErrList", targetNamespace = "http://xml.asyst.co.id/commons/ws/fault")
public class Fault extends Exception {

    private id.co.asyst.bukopin.mobile.user.model.soap.account.ErrList errList;

    public Fault() {
        super();
    }

    public Fault(String message) {
        super(message);
    }

    public Fault(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public Fault(String message, id.co.asyst.bukopin.mobile.user.model.soap.account.ErrList errList) {
        super(message);
        this.errList = errList;
    }

    public Fault(String message, id.co.asyst.bukopin.mobile.user.model.soap.account.ErrList errList, java.lang.Throwable cause) {
        super(message, cause);
        this.errList = errList;
    }

    public id.co.asyst.bukopin.mobile.user.model.soap.account.ErrList getFaultInfo() {
        return this.errList;
    }
}
